pkg/model/deployment.proto

```diff
@@ -88,11 +88,18 @@ message Deployment {
     int64 updated_at = 102 [(validate.rules).int64.gte = 0];
 }
 
+enum SyncStrategy {
+    NONE = 0;
```

Comment:
`NONE` is a bit confusable to me. I feel like it would be better to use `AUTO` or `UNDEFINED` or something like that.

Reply:
Nice. `AUTO` looks good.